methods/show_rentals.rb

```diff
@@ -0,0 +1,31 @@
+require_relative './rentals_memory'
+require_relative './people_memory'
+
+class DisplayRentals
+  include RentalsMemory
+  include PeopleMemory
+
+  def initialize
+    @people = show_people
+    @rentals = show_rentals
+  end
+
+  def show
+    if @rentals.empty?
+      puts "\nAdd a new rental before continue =D"
+    else
+      puts "\nSelect a person by ID from the list: "
+      @people.each do |person|
+        print "#{@people.find_index(person) + 1}) "
+        print "[#{person.class}] ID: #{person.id} Name: #{person.name}\n"
+      end
+      person_selected = gets.chomp.to_i
+      puts 'Rentals: '
+      @rentals.each do |rental|
+        if rental.person.id == person_selected
+          puts "Date: #{rental.date}, Book: #{rental.book.title}, Author: #{rental.book.title}"
```

Comment:
- [ ] There seems to be some repetition in the console output of your app when listing rentals for a person. I think you meant `Author: #{rental.book.author}`

  ![alex-repeat](https://user-images.githubusercontent.com/36542986/152008247-13f1882e-224c-4bb7-9f87-6aca23f8c094.jpg)